Client/Frontend/Browser/URIFixup.swift

```diff
@@ -32,6 +32,18 @@ class URIFixup {
         if trimmed.range(of: " ") != nil {
             return nil
         }
+        
+        // A URL is only valid when the URL has a scheme, is not an email, is not quoted, and can be punycoded.
+        // If one of the above conditions is NOT satisfied, the URL is invalid and should be deemed "search terms" search terms instead. Technically, an email is also a valid URL but does not get handled by the DNS server.
```

Comment:
Can we fix-up these comments to be contained to a max 110 char column count?

Reply:
Fixed.